Makefile

```diff
@@ -39,11 +40,28 @@ lint:
 		exit 1; \
 	fi
 
+.PHONY: ssvsigner-lint
+ssvsigner-lint:
+	cd ssvsigner && $(SSVSIGNER_RUN_TOOL) golangci-lint -c ../.golangci.yaml run -v ./... && cd ..
+	@if [ ! -z "${UNFORMATTED}" ]; then \
+		echo "Some files requires formatting, please run 'go fmt ./...'"; \
+		exit 1; \
+	fi
```

Comment:
A few suggestions to improve integration
```makefile
  .PHONY: lint
  lint:
        $(RUN_TOOL) golangci-lint run -v ./...
        @$(MAKE) ssvsigner-lint

  .PHONY: ssvsigner-lint
  ssvsigner-lint:
        cd ssvsigner && GOWORK=off $(SSVSIGNER_RUN_TOOL) github.com/golangci/golangci-lint/v2/cmd/golangci-lint run -c ../.golangci.yaml -v ./...
```

  Changes:
  1. Added @$(MAKE) ssvsigner-lint to lint target so both run together
  2. Added GOWORK=off (required when using -modfile)
  3. Used full package path for golangci-lint with go run
  4. Removed unnecessary && cd ..
  5. Removed the UNFORMATTED check (it's never set)

  This way make lint runs everything, and the separate workflow files can be deleted!